src/home_robot_hw/home_robot_hw/ros/grasp_helper.py

```diff
@@ -80,7 +80,7 @@ def _cb(self, msg):
         # Convert rospy point cloud into a message that we care about
         with self.grasp_lock:
             self.grasp_id = msg.seq
-            self.poses = msg_to_poses(msg)
+            self.poses = self.msg_to_poses(msg)
```

Comment:
what was happening before? how was this working?

Reply:
There's no way this was working before. We probably just never used this function.